packages/notifi-react-card/lib/components/subscription/NotifiSubscribeButton.tsx

```diff
@@ -106,7 +107,16 @@ export const NotifiSubscribeButton: React.FC<NotifiSubscribeButtonProps> = ({
 
   const renewTargetGroups = useCallback(async () => {
     if (isCanaryActive) {
-      return frontendClient.ensureTargetGroup(targetGroup);
+      let result = {} as TargetGroupFragmentFragment;
+      try {
+        result = await frontendClient.ensureTargetGroup(targetGroup);
+      } catch (e) {
+        // catch the case that the 'Default' discord already created
+        const data = await frontendClient.fetchData();
+        if (!data.targetGroup?.[0]) throw e;
```

Comment:
instead of getting the first targetGroup, we should do a `find`

```
const defaultTargetGroup = data.targetGroup?.find(it => it.name === 'Default');
if (defaultTargetGroup === undefined) {
  throw e;
}

result = defaultTargetGroup;
```